### PR TITLE
101-webapp-linux-managed-mysql - WebApp + MySQL - Update

### PR DESCRIPTION
**Pull request by Krisnatagoras**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by leestott** [comment]
@bmoore-msft  This fails as the MYSQL version is only 2017 this is the latest version https://docs.microsoft.com/en-us/azure/templates/Microsoft.DBforMySQL/2017-12-01/servers so as the API is so old the pipeline fails as the API is older than 2  years but Template deploys.

File: 101-webapp-linux-managed-mysql/azuredeploy.json [hunk 1]
```diff
@@ -99,19 +100,41 @@
...
-                "description": "Azure database for PostgreSQL sku family"
+                "description": "Azure database for mySQL sku family"
             }
         }
     },
     "variables": {
-        "databaseName": "[concat(parameters('siteName'), 'database')]",
-        "serverName": "[concat(parameters('siteName'), 'mysqlserver')]",
-        "hostingPlanName": "[concat(parameters('siteName'), 'serviceplan')]"
+        "databaseName": "[concat('database', uniqueString(resourceGroup().name))]",
```
Comment (bmoore-msft) [inline 101-webapp-linux-managed-mysql/azuredeploy.json]: this does not need to be globally unique, so just use a name appropriate for the workload

File: 101-webapp-linux-managed-mysql/azuredeploy.json [hunk 2]
```diff
@@ -99,19 +100,41 @@
...
+        "serverName": "[concat('mysql-',  uniqueString(resourceGroup().name))]",
```
Comment (bmoore-msft) [inline 101-webapp-linux-managed-mysql/azuredeploy.json]: this needs to be globally unique, so use resourceGroup().id

Comment (Krisnatagoras) [reply 101-webapp-linux-managed-mysql/azuredeploy.json]: Thanks for the feedback @bmoore-msft 

Done.

File: 101-webapp-linux-managed-mysql/azuredeploy.json (hunk 1 above)
Comment (Krisnatagoras) [reply 101-webapp-linux-managed-mysql/azuredeploy.json]: Okay

**Pull request merged**